### DISCUSSION TITLE
What does blocking stats means, it does show busy all the time?

### DISCUSSION THREAD

**Discussion by gftea** [tokio-rs/console#583] [General]
From the blocking task stats, does it mean it is busy all the time for 16h? or simply it is not calculating the stats of blocking tasks?

![image](https://github.com/user-attachments/assets/e102574c-acd1-479c-94de-38107442cd83)

**Comment by hds** [comment]
It's the kind of the task. A blocking task is run on one of the blocking workers, not on the workers shared by async tasks. Blocking tasks never yield to a runtime (they're not async). Currently blocking task running time counts everything as busy, even if it yields the thread or the thread is put to sleep for any reason.
